### PR TITLE
Added neptune-client recipe

### PR DESCRIPTION
**Pull request by szymon-kuklewicz**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by hubertjaworskineptune** [comment]
Hi, as part of Neptune team, I'd like to be a maintainer here

File: recipes/neptune-client/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  summary: "Neptune Client"
+  doc_url: "https://github.com/neptune-ml/neptune-client/blob/master/README.md"
+  dev_url: "https://github.com/neptune-ml/neptune-client"
+
+extra:
+  recipe-maintainers:
+    - szymon-kuklewicz
+    - aniezurawski
+    - hubertjaworskineptune
+    - pitercl
```
Comment (scopatz) [inline recipes/neptune-client/meta.yaml]: @pitercl @aniezurawski are you OK with being maintainers?

File: recipes/neptune-client/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    - tests.neptune.internal.hardware
+    - tests.neptune.internal.hardware.gauges
+    - tests.neptune.internal.hardware.metrics
+    - tests.neptune.internal.hardware.resources
+    - tests.neptune.internal.storage
+    - tests.neptune.utils
+
+about:
+  home: "https://neptune.ai/"
+  license: "Apache Software"
```
Comment (scopatz) [inline recipes/neptune-client/meta.yaml]: Which version?

**Comment by aniezurawski** [comment]
Hi, as part of Neptune team, I'd like to be a maintainer here

**Comment by pitercl** [comment]
Hi, as part of Neptune team, I'd like to be a maintainer here.

**Comment by szymon-kuklewicz** [comment]
@conda-forge-admin, please ping team

**Comment by szymon-kuklewicz** [comment]
Can you help me and explain what is the reason why win_64/appveyor checks fails ? 
I don't see any error there.

File: recipes/neptune-client/meta.yaml (hunk 2 above)
Comment (szymon-kuklewicz) [reply recipes/neptune-client/meta.yaml]: Specified Apache-2.0

File: recipes/neptune-client/meta.yaml (hunk 1 above)
Comment (szymon-kuklewicz) [reply recipes/neptune-client/meta.yaml]: Maintainers confirmed

**Comment by isuruf** [comment]
You need https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

**Comment by szymon-kuklewicz** [comment]
What exactly entry points do and they are needed here since this library is noarch (pure python library) ?

**Comment by szymon-kuklewicz** [comment]
AppVeyor fails due to invalid certificate. Please verify why:

Checking to see if this PR build is outdated.
9Traceback (most recent call last):
10  File "C:\projects\staged-recipes\ff_ci_pr_build.py", line 201, in <module>
11    sys.exit(main())
12  File "C:\projects\staged-recipes\ff_ci_pr_build.py", line 192, in main
13    exit_code = int(appveyor_check_latest_pr_build(repo, pr, bld) is False)
14  File "C:\projects\staged-recipes\ff_ci_pr_build.py", line 117, in appveyor_check_latest_pr_build
15    data = request_json(url.format(repo=repo, total_builds=total_builds), headers=headers)
16  File "C:\projects\staged-recipes\ff_ci_pr_build.py", line 46, in request_json
17    with contextlib.closing(urlopen(request)) as response:
18  File "C:\Python27\lib\urllib2.py", line 154, in urlopen
19    return opener.open(url, data, timeout)
20  File "C:\Python27\lib\urllib2.py", line 429, in open
21    response = self._open(req, data)
22  File "C:\Python27\lib\urllib2.py", line 447, in _open
23    '_open', req)
24  File "C:\Python27\lib\urllib2.py", line 407, in _call_chain
25    result = func(*args)
26  File "C:\Python27\lib\urllib2.py", line 1241, in https_open
27    context=self._context)
28  File "C:\Python27\lib\urllib2.py", line 1198, in do_open
29    raise URLError(err)
30urllib2.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:727)>
31

**Comment by szymon-kuklewicz** [comment]
@isuruf can you explain why do I need entry points since this library is noarch (pure python library)?

**Comment by szymon-kuklewicz** [comment]
@conda-forge-admin, please ping team

**Comment by scopatz** [comment]
Entry-points are the noarch way of adding a command-line utility

**Comment by szymon-kuklewicz** [comment]
I understand but why is this required because without entry point building package for windows platform fails

**Comment by szymon-kuklewicz** [comment]
Can you also help with AppVeyor build? It looks like it's failure is not connected to the feedstock itself but rather is a failure of CI. Am I right ?

**Comment by jakirkham** [comment]
Yeah AppVeyor can be ignored.

**Comment by scopatz** [comment]
Yeah, since it is noarch, only linux is truly needed

**Comment by scopatz** [comment]
Thanks All!

**Pull request merged**